source/quick-start.txt

```diff
@@ -98,6 +98,12 @@ Because you installed the BOM, you can omit a version in the
 {+driver-short+} dependency entry. The version you specify in the
 BOM determines the dependency versions to install.
 
+.. sharedinclude:: dbx/jvm/BOM-installation-tip.rst
+
+   .. replacement:: installation-guide
+
+      :ref:`Installation Guide<kotlin-get-started-install-bom>`
+
```

Comment:
S: since the preceding text already mentions the BOM, you do not need to include the tip here.